pkg/scheduler/framework/status.go

```diff
@@ -0,0 +1,137 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package framework
+
+import "strings"
+
+// StatusCode is the status code of a Status, returned by a plugin.
+type StatusCode int
+
+// Pre-defined status codes.
+const (
+	// Success signals that a plugin has completed its run successfully.
+	// Note that a nil *Status is also considered as a Success Status.
+	Success StatusCode = iota
+	// internalError signals that a plugin has encountered an internal error.
+	// Note that this status code is NOT exported; to return an internalError status, use the
+	// FromError() call.
+	internalError
+	// ClusterUnschedulable signals that a plugin has found that a placement should not be bound
+	// to a specific cluster.
+	ClusterUnschedulable
+	// Skip signals that no action is needed for the plugin to take at the stage.
+	// If this is returned by a plugin at the Pre- stages (PreFilter or PreScore), the associated
+	// plugin will be skipped at the following stages (Filter or Score) as well. This helps
+	// reduce the overhead of having to repeatedly call a plugin that is not needed for every
+	// cluster in the Filter or Score stage.
+	Skip
+)
+
+var statusCodeNames = []string{"Success", "InternalError", "ClusterUnschedulable", "PreSkip"}
```

Comment:
Why are those values not alligned with the state name?

Reply:
My bad. This inconsistency was left out when sending out the PR (it was originally called `PreSkip`). I will correct it.